Makefile

```
@@ -49,9 +49,14 @@ compile-multiarch:
 	$(MAKE) compile GOOS=linux GOARCH=arm
 	$(MAKE) compile GOOS=linux GOARCH=arm64
 
-test:
+test: test-unit
+test-unit:
 	@echo "=== $(INTEGRATION) === [ test ]: Running unit tests..."
-	@go test -race ./...
+	@go test -v -race ./...
+
+test-integration:
+	@echo "=== $(INTEGRATION) === [ test ]: Running integration tests..."
+	@go test -v ./test/integration -tags integration
```

nit: Usually test target is specified as the last argument
```suggestion
	@go test -v -tags integration  ./test/integration
```